ipn/ipnlocal/filesharingstatus_windows.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//+build windows
+
+package ipnlocal
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"golang.org/x/sys/windows/registry"
+)
+
+const (
+	sendFileShellKey = "*\\shell\\tailscale"
+)
+
+var (
```

Comment:
var ipnExePath string // lazily initialized path to "C:\Program Files\Tailscale IPN" etc